README.md

```diff
@@ -145,3 +165,311 @@ Releases are aligned to Vega core [releases](https://github.com/vegaprotocol/veg
+## 9. Set up Development environment
+   ### 1. Install "Remote - WSL" extension
+1. Launch Visual Studio Code on your Windows machine.
+
+2. Open the Extensions view by clicking on the square icon on the left sidebar or by pressing `Ctrl+Shift+X`.
+
+3. In the Extensions view, search for "Remote - WSL" in the search bar.
+
+4. Look for the "Remote - WSL" extension in the search results. It should be the one published by Microsoft with the ID `ms-vscode-remote.remote-wsl`.
+
+5. Click on the "Install" button next to the "Remote - WSL" extension.
+
+6. Visual Studio Code will begin downloading and installing the extension.
+
+7. Once the installation is complete, you will see an "Open" button instead of "Install". Click on the "Open" button to activate the extension.
+
+8. Open your WSL terminal (ensure you are in the vega-market-sim directory). Run the command
+   ```
+   code .
+   ```
+This will open up a visual studio window connected to your linux environment.
+
+__If visual studio is unable to connect to your environment perform these steps to restart your wsl__
+1. Close Visual studio
+2. Run **COMMAND PROMPT** as admin
+
+
+    Find out which instance of WSL is running
+
+      ```
+      wsl -l --running
+      ```
+
+    Terminate all running instances of WSL
+
+   ```
+   wsl --shutdown
+   ```
+
+    Then simply restart WSL
+   ```
+   wsl 
+   ```
+3. Reopen Visual studio
+4. Open your WSL terminal (ensure you are in the vega-market-sim directory). Run the command
+   ```
+   code .
+   ```
+
+
+## OPTIONAL Setup for the UI
+## 1. Install Node to WSL
+To install NVM (Node Version Manager) and Node.js version 16 in Windows Subsystem for Linux (WSL), follow these steps:
+
+1. Install the required dependencies for NVM using the following command:
+
+   ```
+   sudo apt install curl ca-certificates bash
+   ```
+
+2. Download the NVM installation script using the following command:
+
+   ```
+   curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
+   ```
+
+3. Close and reopen your terminal or run the command `source ~/.bashrc` to load the NVM environment variables.
+
+4. Verify that NVM is installed properly by running the following command:
+
+   ```
+   nvm --version
+   ```
+
+5. Install Node.js version 16 using NVM by running the following command:
+
+   ```
+   nvm install 16
+   ```
+
+   This command will download and install Node.js version 16 in your WSL environment.
+
+6. Verify that Node.js is installed successfully by checking its version:
+
+   ```
+   node --version
+   ```
+
+   It should display the version number of Node.js 16.x.
+
+7. To set Node.js 16 as the default version, run the following command:
+
+   ```
+   nvm alias default 16
+   ```
+
+   This will make Node.js version 16 the default version whenever you open a new terminal session.
+## 2. Install Yarn
+To install Yarn on Windows Subsystem for Linux (WSL), you can follow these steps:
```

Comment:
As above, feels like this might be safest just linking to a yarn installation tutorial?